app/models/comment.rb

```diff
@@ -0,0 +1,16 @@
+class Comment < ApplicationRecord
+  belongs_to :user, foreign_key: 'author_id'
+  belongs_to :post
+
+  validates :author_id, presence: true
+  validates :post_id, presence: true
+  validates :text, presence: true
+
+  after_save :update_comments
+
+  def update_comments_counter
+    post = self.post
```

Comment:
- [ OPTIONAL ]  Since you changed the name here, so I highly recommend changing it in line 9 as well 💪🏻💯🔥